src/components/Chat/ChatMessages.tsx

```diff
@@ -0,0 +1,84 @@
+import { Typography } from '@/components/common/Typography';
+import { LoadingSpinner, LoadingSpinnerWrapper } from '@/components/common/LoadingSpinner';
+import { BASE_URL } from '@/constants/routes';
+import {
+  ChatMessagesArea,
+  AssistantMessageContainer,
+  UserMessageContainer,
+  CharacterAvatar,
+  AssistantBubble,
+  UserBubble,
+} from '@/components/Chat/Chat.styles';
+
+type Chat = {
+  role: 'user' | 'assistant';
```

Comment:
별거아니긴한데, 
보통 서버 API Response와 관계있는 타입을 Response 타입을 가져와서 타입을 지정해주는게 더 직관적이고, 
타입이 변경되어도 수정포인트가 적어지고, 발생가능한 버그를 미리 알 수 있답니다!
```suggestion
  role: ChatLog['role'];
```